ansible/roles/ocp4-workload-er-demo/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+map_token:
+release_tag: 2.4
+
+base_dir: /tmp
```

Comment:
All variables (except the 3 common ones) should be prefixed by the role name: ocp4_workload_er_demo_